account_invoice_refund_link/tests/test_invoice_refund_link.py

```diff
@@ -8,8 +8,6 @@
 
 
 class TestInvoiceRefundLinkBase(TransactionCase):
-    refund_method = "refund"
```

Comment:
Removing this you are doing all the tests the same, and the idea is to test all the scenarios.

Reply:
The `refund_method` field in v17 no longer exists, they use `reverse_moves` method with the parameter `is_modify` to determine what to do with the new credit note. What is the best approach to this situation?

Reply:
Then reduce the number of tests, but test both conditions, with and without `is_modify`.